src/router.js

```diff
@@ -281,10 +290,16 @@ const router = new Router({
         path: 'resultats/:droitId',
         component: () => import(/* webpackChunkName: "resultats" */ './views/Simulation/ResultatsDetail.vue'),
       },
+      {
+          path: ':element/en_savoir_plus',
+          name: 'simulation_en_savoir_plus',
+          component: () => import(/* webpackChunkName: "en_savoir_plus" */ './components/EnSavoirPlusContent.vue')
+      },
```

Comment:
Est ce que cette route peut être déclarée en haut ou éviter la duplication avec `individu_en_savoir_plus` ?

Reply:
je vois pas trop comment faire ça... pour moi on est obligé d'en déclarer une par niveau.

Reply:
https://github.com/pillarjs/path-to-regexp/tree/v1.7.0#one-or-more Je suis en train de faire un commit sur la branche.

Reply:
Vu à la TV https://router.vuejs.org/guide/essentials/dynamic-matching.html#advanced-matching-patterns